stable/spinnaker/Chart.yaml

```
@@ -1,7 +1,7 @@
 apiVersion: v1
 description: Open source, multi-cloud continuous delivery platform for releasing software changes with high velocity and confidence.
 name: spinnaker
-version: 1.1.7
+version: 1.1.8
```

adding a new functionality 
```suggestion
version: 1.2.0
```

---

Thanks. I've changed that to 1.2.1.